workers/exclusive-content/README.md

```diff
@@ -0,0 +1,92 @@
+# Exclusive Content Worker
+
+This cloudflare worker will encrypt any plain text given a payment pointer. Furthermore, it will decrypt the resulting cypher text if it can verify payment by a receipt verifier.
+
+## Testing
+
+Configure your `wrangler.toml` file.
+
+```sh
+npm install
+wrangler secret put MASTERKEY
+# enter a key to be stored with cloudflare worker
+wrangler preview --watch
+```
+
+## Deploying
+
+Configure your `wrangler.toml` file.
+
+```sh
+npm install
+wrangler secret put MASTERKEY
+# enter a key to be stored with cloudflare worker
+wrangler publish
+```
+
+## Endpoints
+
+### `POST /encrypt`
+
+Encrypts plaintext
+
+#### Request Body
+
+JSON object containing
+| Name | Type | Description | Example |
+| -- | -- | -- | -- |
+| pp | String | (proxy) payment pointer | "\$spsp.example.com/alice" |
+| pt | String | plain text | "Hello World" |
+
+#### Headers
+
+| Header       | Value            |
+| ------------ | ---------------- |
+| Content-Type | application/json |
+
+#### Response Body
+
+JSON object containing
+| Name | Type | Description | Example |
+| -- | -- | -- | -- |
+| ct | String | cypher text | "VsPD81db5Ts9rUnvxqGD5hxCRoDU+ygE44w42AoEbaKQbhGXkrpxoYwCrhrtR2MmFfOg" |
+| iv | String | initialization vector | "1603118464654" |
```

Comment:
what encoding is this? should be noted for all fields that represent binary data

Reply:
It is an array buffer that was converted to a utf-8 string and then base64 encoded. How detailed does it have to be?

Reply:
if we say that it's base64 encoded then that's fine; it just wasn't clear as the iv appears to be all numbers

Reply:
Oh sorry, I didn't look at this carefully. The iv is just a random number, converted to string. I can make that alphanumeric like the nonce, too.